db/migrate/20200718131012_create_va_caution_flags.rb

```diff
@@ -0,0 +1,17 @@
+
+class CreateVaCautionFlags < ActiveRecord::Migration[4.2]
+    def change
+      create_table :va_caution_flags do |t|
+        t.string :institution_id, null: false
```

Comment:
should this be called `facility_code` so as to not confuse it with be a foreign key column that relates to the `institutions` table?

Reply:
I changed it.